### PR TITLE
added line in find_file_path to look in plugins/<plugin>/data/ directory for files

### PR DESCRIPTION
**Pull request by garunagiri**
Payloads that are compiled into plugins/<plugin>/data/payloads should now be found, as well as any other files in the plugins/<plugin>/data/ directory.

### CODE REVIEW

File: app/service/file_svc.py [hunk 1]
```diff
@@ -73,6 +73,9 @@ def __init__(self, plugins, exfil_dir):
             file_path = await self._walk_file_path('plugins/%s/%s' % (plugin, location), name)
             if file_path:
                 return plugin, file_path
+            file_path = await self._walk_file_path('plugins/%s/data/%s' % (plugin, location), name)
```
Comment (wbooth) [inline app/service/file_svc.py]: iterate through list instead of calling twice.

also, shouldn't walk file path "walk" it and find in the sub directory?

Comment (wbooth) [reply app/service/file_svc.py]: I think _walk_file_path maybe the issue here

Comment (garunagiri) [reply app/service/file_svc.py]: The issue is that data/plugins not a sub directory of the given path, it's another parallel directory. Unless I'm understanding you wrong

Comment (garunagiri) [reply app/service/file_svc.py]: Changed to iterate through a for loop. But it's still getting called twice, is this what you wanted?

File: app/service/file_svc.py [hunk 2]
```diff
@@ -70,9 +70,10 @@ def __init__(self, plugins, exfil_dir):
         :return: a tuple: the plugin the file is found in & the relative file path
         """
         for plugin in self.plugins:
-            file_path = await self._walk_file_path('plugins/%s/%s' % (plugin, location), name)
-            if file_path:
-                return plugin, file_path
+            for subd in ['', 'data/']:
+                file_path = await self._walk_file_path('plugins/%s/%s%s' % (plugin, subd, location), name)
```
Comment (wbooth) [inline app/service/file_svc.py]: remove slash from subdirectory name and I'd recommend using pathlib or sys to concatenate paths

**Comment by wbooth** [comment]
@garunagiri please resolve the merge conflict.

**Comment by garunagiri** [comment]
@wbooth conflict resolved

**Pull request merged**